### PR TITLE
fix: added getDefaultLocale function

### PR DESCRIPTION
**Pull request by gazpachu**
Fixes FL-458

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -21,7 +21,18 @@ import { Locale, LocaleCode } from './types';
 
 export * from './types';
 
-export const DEFAULT_LOCALE_CODE = LocaleCode.English;
+export function getDefaultLocale(selectableLocales?: LocaleCode[]) {
```
Comment (wahidmagdy) [inline src/index.ts:24]: LGTM, but i'd prefer this method to be written after `ordered` so it makes sense to use something after it's definition :sweat_smile:

Comment (gazpachu) [reply src/index.ts]: done

File: src/index.ts [hunk 2]
```diff
@@ -47,6 +45,22 @@ export const ordered: Locale[] = [
   arabic
 ];
 
+/**
+ * Returns the correct default locale
+ */
+export function getDefaultLocale(selectableLocales?: LocaleCode[]) {
+  const locales: Locale[] = selectableLocales ? ordered.filter((item: Locale) =>
```
Comment (wahidmagdy) [inline src/index.ts:52]: are `selectableLocales` ordered by the backend automatically or do shops select the specific locale order?

Comment (gazpachu) [reply src/index.ts:52]: Interesting question. I don't think currently there's a way to order the locales

**Review by wahidmagdy — APPROVED** [review]
LGTM

File: src/index.ts [hunk 3]
```diff
@@ -47,6 +45,22 @@ export const ordered: Locale[] = [
...
+      selectableLocales.find((locale) => locale === item.code)) : ordered;
+
+  const englishLocale = locales.find(locale => locale.code === LocaleCode.English);
+  if (englishLocale) return englishLocale;
+
+  const japaneseLocale = locales.find(locale => locale.code === LocaleCode.Japanese);
+  if (japaneseLocale) return japaneseLocale;
```
Comment (Findmitrey) [inline src/index.ts:59]: It seems to me better to provide the ability to set the default language based on `selectableLocales` and sort it by order to find first available locale: 
```
const locales = .... ;

return locales[0]
```

Comment (wahidmagdy) [reply src/index.ts:59]: That was what i was thinking, but it requires the backend to return the locales sorted each time. That may not be the case, or perhaps the shop can select the order of the locales as well. both of these would be issues so it's safest to search for the default locales first

**Review by Findmitrey — APPROVED** [review]

File: src/index.ts (hunk 2 above)
Comment (SimeonC) [reply src/index.ts:52]: I think I have a sort locales until function in settings I could move over. Was created just for this case of "if it isn't ordered how I want it to be"

**Pull request merged**